addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -933,7 +933,7 @@ func _get_valid_oriented_animation(animation_label):
 		if animation_player.has_animation(animation):
 			return animation
 
-	return null
+	return "null"
```

Comment:
```suggestion
	return ""
```

Additionally, the signature of the method should also be updated to include the returning type: `func _get_valid_oriented_animation(animation_label: String) -> String:`